.gitignore

```diff
@@ -6,3 +6,4 @@ bower_components
 Gemfile.lock
 styleguide/css/*.css
 backstop_data/bitmaps_test
+npm-debug.log
```

Comment:
The same could be said about most of the files in the `.gitignore` file. It's better to not allow it into the app via the repo's `.gitignore` than assuming every contributor will have added this to the their global gitignore file.